types/tuya-panel-kit/index.d.ts

```diff
@@ -11323,7 +11323,7 @@ export interface NavigationComponentClass<P = {}, S = {}> {
     new (props: P, context?: any): NavigatorLayout<P, S>;
 }
 // tslint:disable-next-line no-unnecessary-generics
-export function createNavigator<P = {}, S = {}>(createNavigatorParam: NavigationParam): NavigationComponentClass<P, S>;
+export function createNavigator<P = {}, S = {}>(createNavigatorParam: NavigationParam, navigationContainerProps: Omit<NavigationContainerProps, 'children'>): any;
```

Comment:
I'm surprised to see a new required parameter without a version bump.  Maybe this was intended to be optional?  And what happened to the return type?

Reply:
Thanks for review, I have bump the version and set new parameter as optional.